src/types/user.ts

```diff
@@ -15,6 +15,7 @@ export interface User {
   gamesPlayed?: number
   colorMode?: "light" | "dark"
   settings?: UserSettings
+  isAnonymous?: boolean
 }
 
-export interface UserSettings {}
+export interface UserSettings { }
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/albertnahas/Lingwars/pullRequest?prid=9585773)
```suggestion
export interface UserSettings {}
```